Assets/Scripts/Actors/ActorComponents/ActorPhysics.cs

```diff
@@ -122,6 +123,12 @@ public float rollMoveSpeed
 	Vector3 _climbSurfaceUp = Vector3.zero;
 	#endregion
 
+	#region Gliding
+	[Space( 10 ), Header( "Gliding" )]
+	[SerializeField] float _glideHorizontalSpeed = 5.0f;
+	[SerializeField] float _glideDescentRate = 1.0f;
```

Comment:
Double check these default values. They should either be completely usable defaults or completely unusable. No use in making defaults that could be seen as okay and aren't.